wdl-parsing/WDLParsingFunction/ignore.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<FindBugsFilter>
```

Comment:
code comment, why is this needed and why is it ok?

Reply:
I'll add a code comment, but it appears it's because they're generated files